### ISSUE TITLE
Issue with running install on macOS

### ISSUE THREAD

**Issue by riaanduplessis** [jlisher/app-helper#1]
### After cloning the repository and entering my project folder:

Ran the command: `./<system-path>/app-helper/src/app.sh install`

This results in a not found error: `no such file or directory`

Then using _sh_: `sh ./<system-path>/app-helper/src/app.sh install`

### Output

```sh
/<system-path>/app-helper/src/app.sh: line 4: realpath: command not found
/<system-path>/app-helper/src/app.sh: line 8: ./variables.sh: No such file or directory
```

### OS

macOS Big Sur - Version 11.2.3

**Comment by jlisher** [comment]
I thought there would be issues. 

I am actually busy rewriting it for sh compatible and adding a build feature so there is no need to source files. 

These changes should make it more stable across platforms.

**Comment by jlisher** [comment]
Oh yeah, I just looked at this issue again now. 

I think if you install `realpath` (a utility to handle parsing file paths). I thought it was standard.

**Comment by riaanduplessis** [comment]
Okay, will try that and let you know.

A workflow to test it across platforms could perhaps be beneficial?

**Comment by jlisher** [comment]
Yeah, been a bit busy, but looking to make it a lot more flexible and testable.

**Comment by riaanduplessis** [comment]
> Oh yeah, I just looked at this issue again now.
> 
> I think if you install `realpath` (a utility to handle parsing file paths). I thought it was standard.

So it seems there is no native way to install `realpath` on macOS. Have to use something like MacPorts but that isn't really ideal.

**Comment by jlisher** [comment]
@riaanduplessis is `readlink` available on macOS by default?

**Comment by riaanduplessis** [comment]
> @riaanduplessis is `readlink` available on macOS by default?

Yes, although, some posts that popped up about `readlink` and macOS is about the `-f` option, so I thought I should just mention that the `-f` option isn't available on macOS in case it makes any difference.

**Comment by jlisher** [comment]
Ok cool, will use a `cd` and `pwd` workaround instead. It seems to be the most stable across platforms.

**Comment by jlisher** [comment]
@riaanduplessis please can you test the PR to make sure it is functional before I merge it. The changes should be available on the `bug/remove-realpath` branch.
